src/dattobd.h

```diff
@@ -77,6 +77,15 @@ struct dattobd_info{
 	char bdev[PATH_MAX];
 };
 
+struct dattobd_version {
+    char version_string[20]; // null terminated string for returning version number, allow for a date.
+};
+
+struct dattobd_active_device_info {
+    int count; // in/out  in = max to supply, out = number of info structs returned
+    struct dattobd_info first; // we return an array of these back to back
+};
```

Comment:
this was modeled after fiemap since I happened to be looking at it recently. I was going to put version in here but the ticket was written as two separate items.